<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DontNeedADOClass</class>
 <widget class="QMainWindow" name="DontNeedADOClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>702</width>
    <height>698</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Don't Need ADO !</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <property name="spacing">
     <number>0</number>
    </property>
    <item row="0" column="0" colspan="2">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Working directory</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lEdtWorkingDir"/>
      </item>
      <item>
       <widget class="QPushButton" name="pBtnEdit">
        <property name="maximumSize">
         <size>
          <width>30</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>...</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="0" colspan="2">
     <widget class="QSplitter" name="splitter">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QListWidget" name="listWidget"/>
      <widget class="QStackedWidget" name="stackedWidget">
       <widget class="QWidget" name="page"/>
       <widget class="QWidget" name="page_2"/>
      </widget>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QPushButton" name="pBtnExecutePiplineWithoutTag">
      <property name="text">
       <string>Execute Pipline without tag</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QPushButton" name="pBtnExecutePipline">
      <property name="text">
       <string>Execute Pipline</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>702</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuPipline">
    <property name="title">
     <string>Pipline</string>
    </property>
    <addaction name="actionNew_pipline"/>
    <addaction name="actionLoad_pipline"/>
   </widget>
   <addaction name="menuPipline"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionDelete_Step"/>
   <addaction name="actionAdd_GIT_Clone_Step"/>
   <addaction name="actionAdd_CMake_Step"/>
   <addaction name="actionAdd_Replace_String"/>
   <addaction name="actionAdd_Compile_Step"/>
   <addaction name="actionAdd_Nuget_Step"/>
   <addaction name="actionAdd_Tag_Step"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionNew_pipline">
   <property name="text">
    <string>New pipline</string>
   </property>
  </action>
  <action name="actionLoad_pipline">
   <property name="text">
    <string>Load pipline</string>
   </property>
  </action>
  <action name="actionAdd_GIT_Clone_Step">
   <property name="text">
    <string>Add GIT Clone Step</string>
   </property>
  </action>
  <action name="actionDelete_Step">
   <property name="text">
    <string>Delete Step</string>
   </property>
  </action>
  <action name="actionAdd_CMake_Step">
   <property name="text">
    <string>Add CMake Step</string>
   </property>
  </action>
  <action name="actionAdd_Replace_String">
   <property name="text">
    <string>Add Replace String</string>
   </property>
  </action>
  <action name="actionAdd_Compile_Step">
   <property name="text">
    <string>Add Compile Step</string>
   </property>
  </action>
  <action name="actionAdd_Nuget_Step">
   <property name="text">
    <string>Add Nuget Step</string>
   </property>
  </action>
  <action name="actionAdd_Tag_Step">
   <property name="text">
    <string>Add Tag Step</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="DontNeedADO.qrc"/>
 </resources>
 <connections/>
</ui>
